plugins/nlp/nlp.hpp

```diff
@@ -35,4 +35,9 @@
 #include "nlp_eng.hpp"
 #include <sstream>
 
+extern "C"
+{
+	ECL_NLP_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb);
+	ECL_NLP_API void setPluginContext(IPluginContext * _ctx);
```

Comment:
setPluginContext in the .cpp file seems to be defined as a C++ function in the nlp namespace.  Should that function be moved outside the namespace?